src/components/main/TableToolbar.js

```diff
@@ -53,63 +53,66 @@ DefaultActions.propTypes = {
   selectedIds: PropTypes.arrayOf(PropTypes.string).isRequired,
 };
 
-const TableToolbar = ({
-  numSelected,
-  tableTitle,
-  selected,
-  headerElements,
-  actions,
-}) => {
+const TableToolbar = ({ numSelected, selected, actions }) => {
   const classes = useToolbarStyles();
   const { t } = useTranslation();
   const renderActions = actions ?? DefaultActions;
 
   return (
-    <Toolbar
-      className={clsx(classes.root, {
-        [classes.highlight]: numSelected > 0,
-      })}
-    >
+    <>
       {numSelected > 0 ? (
-        <Typography
-          className={classes.title}
-          color="inherit"
-          variant="subtitle1"
-          component="div"
+        <Toolbar
+          className={clsx(classes.root, {
+            [classes.highlight]: numSelected > 0,
+          })}
         >
-          {t('nbitem selected', { numSelected })}
-        </Typography>
-      ) : (
-        <>
           <Typography
             className={classes.title}
-            variant="h6"
-            id="tableTitle"
+            color="inherit"
+            variant="subtitle1"
             component="div"
           >
-            {tableTitle}
+            {t('nbitem selected', { numSelected })}
           </Typography>
 
-          {headerElements}
-        </>
-      )}
+          {/* {numSelected > 0 ? (
+            <Typography
+              className={classes.title}
+              color="inherit"
+              variant="subtitle1"
+              component="div"
+            >
+              {t('nbitem selected', { numSelected })}
+            </Typography>
+          ) : (
+            <>
+              <Typography
```

Comment:
remove comments?

Reply:
Sorry. I removed this. I'll check the other files too.